pkg/controllers/deprovisioning/types.go

```diff
@@ -121,19 +121,31 @@ func (o Command) String() string {
 			fmt.Fprintf(&buf, "/%s", capacityType)
 		}
 	}
-	if o.replacementNode != nil {
-		ct := o.replacementNode.Requirements.Get(v1alpha5.LabelCapacityType)
+	odNodes := 0
+	spotNodes := 0
+	for _, node := range o.replacementNodes {
+		ct := node.Requirements.Get(v1alpha5.LabelCapacityType)
+		if ct.Has(v1alpha5.CapacityTypeOnDemand) {
+			odNodes++
+		}
+		if ct.Has(v1alpha5.CapacityTypeSpot) {
+			spotNodes++
+		}
+	}
+	// Print list of instance types for the first replacementNode.
+	if len(o.replacementNodes) > 1 {
+		fmt.Fprintf(&buf, " and replacing with %d spot and %d on-demand nodes from types %s",
+			spotNodes, odNodes,
+			scheduling.InstanceTypeList(o.replacementNodes[0].InstanceTypeOptions))
+	} else if len(o.replacementNodes) == 1 {
```

Comment:
It feels weird to me here that we have separate handling for the 1 node case; why can't we use the same string statement for both?

Reply:
I agree it's a little awkward. I thought it would be redundant to just have it print something like ` and replacing with 0 spot and 1 on-demand nodes from types ...` 

Changed it so that it looked a little nicer in code. LMK what you think.